composer.json

```diff
@@ -16,7 +16,7 @@
     "minimum-stability" : "dev",
     "require": {
         "php": ">=7.4",
-        "commongateway/corebundle": ">=1.0.107",
+        "commongateway/corebundle": ">=1.1.19",
```

Comment:
Did you test this with the latest versions of the CoreBundle?
Or is xxllnc Gateway using a specific CoreBundle version 1.1.19 and not higher?